docs/FAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: "FAQ"
+sidebar_position: 13
+---
+
+## Smart accounts and SDK
+
+**Will I get the same smart account address on all chains?**
+
+Yes, you will get the same address on all chains as long as the owner's address is the same.
+    
+**How can I create another smart account using the same EOA on the same chain?**
+    
+you can pass the `index` in the config while creating the account as mentioned in the createSmartAccountClient [method](https://docs.biconomy.io/Account/methods#createsmartaccountclient). First smart account gets created with index 0 by default. 
+    
+**Where can I find more examples of SDK integrations?**
+    
+https://github.com/bcnmy/sdk-examples
+    
+**How can I increase the timeout in the waitForTransactionHash function?**
+    
+Initialize the bundler with updated configuration, to increase the timeout (milliseconds).
+
+```jsx
+import { createSmartAccountClient, createBundler } from "@biconomy/account";
+
+const biconomySmartAccount = await createSmartAccountClient({
+    signer,
+    bundler: await createBundler({
+    bundlerUrl: "bundler url",
+    userOpReceiptMaxDurationIntervals: { 80001: 60000 },
+    }),
+});
+```
+    
+**Which social login methods are supported?**
+    
+Explore services such as Web3Auth, Particle Network, Magic.link, or Lit Protocol to link a user's social identity with a cryptographic key. This key can then serve as the owner of the Smart Account. Be sure to check out various [signers](https://docs.biconomy.io/Account/signers/) integrations for this purpose.
+    
+**How do I enable session keys for users?**
+    
+Utilize the session key validation [module](https://docs.biconomy.io/Modules/sessionvalidationmodule) 
+    
+## Bundler & Paymaster
+
+**Is the bundler URL the same for all chains?**
+    
+Yes, the bundler URL is same for all testnets. For mainnet you can reach out to us.
+    
+**How can I sign the userOp manually?**
+    
+To sign the userOp manually, first, calculate the userOpHash and then sign it using the same signer, the account is then created. you can find more details [here](https://hackmd.io/@xWzOEjWIRIKP22EKSdIgEg/Hk5SGcK-o).
+    
+**Can I just use Biconomy’s bundler with another smart account implementation?**
+    
+Yes, Biconomy's bundler seamlessly integrates with other smart account providers. As long as the user operation remains valid and complies with ERC 4337 specifications, the integration is possible.
+
+**Which ERC20 tokens are supported by the biconomy paymaster? How can I add support for a custom ERC20 token?**
+    
+You can check out the list of supported networks under the [paymaster](/Paymaster/supportedNetworks) and [bundler](/Bundler/supportedNetworks) sections. You can reach out to us here for custom token requirements.
```

Comment:
Could mention smartAccount.getSupportedTokens()  here

Reply:
is this a new method in smartAccount?, yes we can mention here and add in methods as well.

Reply:
@joepegler let me know once you add this method in the sdk, will add on this change in a new PR.